### ISSUE TITLE
Choose R Installation Dialog focus location not always reported by NVDA or Narrator

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6407]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.887
    OS Version      : Windows-10
    R Version       : None

### Steps to reproduce the problem
- Remove (or never install) R
- Move mouse cursor off screen (e.g. to a corner) or disable mouse-tracking in NVDA so it isn't a factor
- Start NVDA (2019.3.1)
- Configure to show Focus highlights (for sighted users)
    - NVDA+Ctrl+G
    - Move down to Vision settings
    - Enable Highlighted, and all three highlight options
- Run RStudio via keyboard
- Because R is absent, get the Choose R Installation dialog box instead of the RStudio main window
- The dialog title is announced, then the selected radio button ("Use your machine's default version of R64 (64-bit)"), the NVDA cursor (blue rectangle) is around that radio button
- Use Tab key to move through the dialog

### Describe the problem in detail
- Sometimes it works, but sometimes the NVDA Cursor remains on that initial radio button, even as keyboard focus moves through the dialog (can see the focus indicators on the other controls as they get focus); nothing is announced about the controls getting focus
- When this happens, the dialog is unusable via screen reader
- Cannot figure out what makes it work sometimes and not other times

### Describe the behavior you expected
- Should always announce the control receiving focus, and should advance the NVDA cursor to same location

### Other Notes
- JAWS seems to handle this dialog, haven't been able to repro using JAWS 2020
- Windows-10 Narrator NEVER announces anything in this dialog

Likely related:
https://bugreports.qt.io/browse/QTBUG-69472
https://github.com/nvaccess/nvda/issues/8604
